src/SumoCoders/FrameworkCoreBundle/Resources/views/Form/fields.html.twig

```diff
@@ -106,9 +108,11 @@
       {% set day_widget = form_widget(form.day, {'attr': {'class': attr.widget_class|default('') ~ 'day'}, 'horizontal_input_wrapper_class': horizontal_input_wrapper_class|default('col-sm-3')}) %}
 
       {{ date_pattern|replace({
+      '/' : '',
```

Comment:
this seems a bit a strange solution, what exactly is it you are doing here?

Reply:
fixed 2d1d468
this fixes the layout when selects are used instead of a datepicker